src/test/resources/spec/concordion/annotation/Resources.html

```diff
@@ -0,0 +1,218 @@
+    	</p>
+    </div> 
+    
+	<!-- ============================================= -->
+	<div class="example">
+        <h3>Example: Multiple Resources, Types and Locations</h3>
+		
+		<p>Executing the following fixture:</p>
+		
+		<pre class="java" concordion:set="#javaFragment">
+import org.concordion.api.Resources;
+import org.concordion.api.Resources.InsertType;
+import org.concordion.integration.junit4.ConcordionRunner;
+import org.junit.runner.RunWith;
+
+@RunWith(ConcordionRunner.class)
+@Resources( value = { "demo.css", "/spec/demo.js", "/spec/demo.txt" }, includeDefaultStyling = false )
+public class ResourcesDemoTest {
+    
+}
+		</pre>
+	
+    	<p concordion:execute="#result1 = process(#javaFragment)">will include 
+    		a link to the resource <span concordion:assertEquals="getLink(#TEXT)">demo.css</span>,
+    		a script link for <span concordion:assertEquals="getScript(#TEXT)">demo.js</span>,
+    		and will copy the resource <span concordion:assertTrue="isCopied(#TEXT)">spec/demo.txt</span>,
+    	</p>
+    </div> 
+    
+    <!-- ============================================= -->
+	<div class="example">
+        <h3>Example: Missing Resource File Throws Exception</h3>
+		
+		<p>Executing the following fixture:</p>
+		
+		<pre class="java" concordion:set="#javaFragment">
+import org.concordion.api.Resources;
+import org.concordion.api.Resources.InsertType;
+import org.concordion.integration.junit4.ConcordionRunner;
+import org.junit.runner.RunWith;
+
+@RunWith(ConcordionRunner.class)
+@Resources( value = "im-not-here.txt" )
+public class ResourcesDemoTest {
+    
+}
+		</pre>
+	
+    	<p>will throw an exception containing the text "<span concordion:assertTrue="processExpectingException(#javaFragment, #TEXT)">No file found</span>".</p>
+    </div> 
+
+    <!-- ============================================= -->
+	<div class="example">
+        <h3>Example: Parent Annotations Are Processed</h3>
+		
+		<p>Executing the following fixture:</p>
+		
+		<pre class="java" concordion:set="#javaFragment">
+import org.concordion.api.Resources;
+
+@Resources( "/spec/demo.js" )
+public class ResourcesDemoTest extends ResourcesDemoParent {
+    
+}
+		</pre>
+		
+		<p>And parent class:</p>
+		<pre class="java" concordion:set="#javaFragmentParent">		
+import org.concordion.api.Resources;
+import org.concordion.integration.junit4.ConcordionRunner;
+import org.junit.runner.RunWith;
+
+@RunWith(ConcordionRunner.class)
+@Resources( value = "demo.css", includeDefaultStyling = false )
+public class ResourcesDemoParent {
+    
+}
+		</pre>
+    	
+		<p concordion:execute="#result1 = process(#javaFragment, #javaFragmentParent)">will include 
+    		a link to the resource <span concordion:assertEquals="getLink(#TEXT)">demo.css</span>,
+    		a script link for <span concordion:assertEquals="getScript(#TEXT)">spec/demo.js</span>,
+    		and will copy the resource <span concordion:assertTrue="isCopied(#TEXT)">spec/demo.txt</span>,
```

Comment:
And maybe add:

and remove the <span concordion:assertFalse="isDefaultCssIncluded()">default CSS</span> from the generated specification.